build/Version.props

```diff
@@ -2,8 +2,8 @@
   <PropertyGroup>
     <VersionMajor>2</VersionMajor>
     <VersionMinor>0</VersionMinor>
-    <VersionPatch>0</VersionPatch>
-    <ReleaseSuffix Condition=" '$(ReleaseSuffix)' == '' ">preview3</ReleaseSuffix>
+    <VersionPatch>1</VersionPatch>
+    <ReleaseSuffix Condition=" '$(ReleaseSuffix)' == '' ">servicing</ReleaseSuffix>
```

Comment:
I don't believe we have used this branding before. Why not preview1? Anyway, is this what the runtime (core-setup) is doing? We should align our branding with theirs.

@Petermarcu